projects/step-frontend/src/lib/angularjs/partials/execution/executionError.html

```diff
@@ -1,4 +1,15 @@
 <div class="panel-body">
+
+  <step-execution-errors
+    [(selected-error-distribution-toggle)]="selectedErrorDistirbutionToggle"
+    [count-by-error-msg]="countByErrorMsg"
+    [count-by-error-code]="countByErrorCode"
+    [error-distribution]="errorDistribution"
+    (search-step-by-error)="searchStepByError($event)"
+  >
+  </step-execution-errors>
+
+<!--
```

Comment:
Not a fan of commenting out code, why is this needed?

Reply:
I left it, as it was left in other templates too.
If you think that it should be removed. Lest remove